chef/cookbooks/crowbar/files/default/crowbar.suse

```diff
@@ -114,14 +114,14 @@
 
 # Check for missing binaries (stale symlinks should not happen)
 # Note: Special treatment of stop for LSB conformance
-RAINBOWS_BIN=/usr/bin/rainbows
-test -x $RAINBOWS_BIN || { echo "$RAINBOWS_BIN not installed"; 
+PUMA_BIN=/usr/bin/puma
```

Comment:
I think for this init script overall https://github.com/ClusterLabs/hawk/blob/master/scripts/hawk.suse.in is a pretty good example to clean it up